docs/reference/search/profile.asciidoc

```diff
@@ -163,7 +163,38 @@ The API returns the following result:
             ]
           }
         ],
-        "aggregations": []
+        "aggregations": [],
+        "fetch": {
+          "type": "fetch",
+          "description": "fetch",
+          "time_in_nanos": 660555,
```

Comment:
This is the actual `nanoTime` for the entire fetch phase rather than the sum of the breakdown. Because we can do that in fetch. We couldn't really do that for aggs or queries.